EKF/sideslip_fusion.cpp

```diff
@@ -193,36 +170,46 @@ void Ekf::fuseSideslip()
 			_innov_check_fail_status.flags.reject_sideslip = false;
 		}
 
+		// Observation Jacobians
+		SparseVector24f<0,1,2,3,4,5,6,22,23> Hfusion;
+		Hfusion.at<0>() = HK21*HK22;
+		Hfusion.at<1>() = HK22*HK25;
+		Hfusion.at<2>() = HK22*HK26;
+		Hfusion.at<3>() = -HK22*HK27;
+		Hfusion.at<4>() = -HK29;
+		Hfusion.at<5>() = HK16*(HK18 - HK30);
+		Hfusion.at<6>() = HK22*HK31;
+		Hfusion.at<22>() = HK29;
+		Hfusion.at<23>() = HK16*HK32;
+
+		// Calculate Kalman gains
+		Vector24f Kfusion;
+		if (!update_wind_only) {
+
+			Kfusion(0) = HK38*HK52;
+			Kfusion(1) = HK50*HK52;
+			Kfusion(2) = HK49*HK52;
+			Kfusion(3) = HK51*HK52;
+			Kfusion(4) = HK48*HK52;
+			Kfusion(5) = HK47*HK52;
+			Kfusion(6) = HK40*HK52;
+
+			for (unsigned row = 7; row <= 21; row++) {
+				Kfusion(row) = HK52*(HK28*P(row,22) - HK28*P(4,row) + HK32*P(row,23) - HK32*P(5,row) + HK33*P(6,row) + HK34*P(2,row) + HK35*P(1,row) - HK36*P(3,row) + HK37*P(0,row));
+			}
+
+		}
+
+		Kfusion(22) = HK45*HK52;
+		Kfusion(23) = HK42*HK52;
+
+		// synthetic sideslip measurement sample has passed check so record it
+		_time_last_beta_fuse = _time_last_imu;
```

Comment:
`_time_last_beta_fuse` is going to be updated once the numerical health checks are passing.

Reply:
Fixed